qiita_db/analysis.py

```diff
@@ -508,6 +508,29 @@ def mapping_file(self):
             return join(base_fp, mapping_fp[0][0])
 
     @property
+    def tgz(self):
```

Comment:
Can you use the function `retrieve_filepaths` from [here](https://github.com/biocore/qiita/blob/master/qiita_db/util.py#L648) ? This way it will look up the mountpoint correctly. This will fail, for example, in the test environment.